RepoCleanup/Functions/CreateOrgWithTeams.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using RepoCleanup.Models;
+using RepoCleanup.Services;
+
+namespace RepoCleanup.Functions
+{
+    public static class CreateOrgWithTeams
+    {
+
```

Comment:
Stylecop says that opening bracets should not be followed by blank line

Reply:
Fixed